### PR TITLE
Make snippet bundle flag a setting, remove waffle.

### PR DESCRIPTION
**Pull request by glogiotatidis**
One less middleware, one less db hit, one less dependency. Good times!

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: snippets/settings.py [hunk 1]
```diff
@@ -171,6 +165,8 @@
     'default': config('CACHE_URL', default='locmem://', cast=django_cache_url.parse),
 }
 
+SERVE_SNIPPET_BUNDLES = config('SERVE_SNIPPET_BUNDLES', default=False, cast=bool)
```
Comment (jgmize) [inline snippets/settings.py]: Wouldn't it make more sense to default this to either `True` or `not DEBUG`?

**Comment by jgmize** [comment]
r+wc, nice work. If you decide to leave the default as-is we just need to make sure that staging and production have the new `SERVE_SNIPPET_BUNDLES` environment variable.

File: snippets/settings.py (hunk 1 above)
Comment (glogiotatidis) [inline snippets/settings.py]: I like `not DEBUG`. /me changes default

**Comment by glogiotatidis** [comment]
> r+wc, nice work. If you decide to leave the default as-is we just need to make sure that staging and production have the new SERVE_SNIPPET_BUNDLES environment variable.

ACK

**Pull request merged**